tools/pcmk_simtimes.in

```diff
@@ -117,13 +117,20 @@ if __name__ == "__main__":
             if result is not None:
                 results = results + [ result ]
 
-        # Sort and print
+        # Sort and print test differences
         results.sort(reverse=args.reverse, key=sort_diff)
         for result in results:
             print("%-40s %6.2fs vs %6.2fs (%+.2fs = %+6.2f%%)" % (result['test'],
                   result['before'], result['after'], result['diff'],
                   result['percent']))
 
+        # Print average differences
+        diff_total = sum(d['diff'] for d in results)
+        percent_total = sum(d['percent'] for d in results)
+        nresults = len(results)
```

Comment:
Is it possible for results to be [] here?  If so, we'll get division by zero on the next line.

Reply:
I think it could if someone set the threshholds high enough not to catch any tests ... I'll add a short-circuit early on.